great_expectations/data_context/data_context/cloud_data_context.py

```diff
@@ -88,6 +87,13 @@
 logger = logging.getLogger(__name__)
 
 
+# TODO Migrate to BaseSettings and support conffile
+class GxCloudConfigEnvVars(BaseModel):
+    GX_CLOUD_BASE_URL: Optional[HttpUrl] = CLOUD_DEFAULT_BASE_URL
+    GX_CLOUD_ACCESS_TOKEN: SecretStr
+    GX_CLOUD_ORGANIZATION_ID: UUID
```

Comment:
TODO add test for non-UUID

Reply:
To access the `SecretStr` value you need to use `.get_secret_value()`
https://docs.pydantic.dev/1.10/usage/types/#secret-types
```python
config = GxCloudConfigEnvVars(gx_cloud_access_token="my_secret", gx_cloud_organization_id=uuid4())

print(config.gx_cloud_access_token)
>>> *******
print(config.gx_cloud_access_token.get_secret_value())
>>> my_secret
```